acceptance/acceptance_test.go

```diff
@@ -275,6 +288,7 @@ func BuildCLI(t *testing.T, cwd, coverDir string) string {
 	cmd = exec.Command(execPath, "--version")
 	out, err = cmd.CombinedOutput()
 	require.NoError(t, err, "%s --version failed: %s\n%s", execPath, err, out)
+	t.Logf("%s --version: %s", execPath, out)
```

Comment:
Why log this? It'll always be a local build, so the output is ~bogus.

Reply:
I was curious, but not sure we need to carry it forward. Removed.